psi4/src/psi4/libmints/dimension.cc

```diff
@@ -105,11 +105,11 @@ Dimension& Dimension::operator-=(const Dimension& b) {
     return *this;
 }
 
-PSI_API bool operator==(const Dimension& a, const Dimension& b) { return (a.blocks_ == b.blocks_); }
+bool operator==(const Dimension& a, const Dimension& b) { return (a.blocks_ == b.blocks_); }
```

Comment:
Why remove the `PSI_API` marker? I think it's necessary to get these overloads visible from outside `core.so`.

Reply:
There is no `PSI_API` in the declaration of the method, in the header file. MSVC errors when signatures do not match. I avoided to change the header, so changed the source. Should I do opposite?

For symbol visibility, on Linux, everything is visible by default. On Windows, I have configure linker to do the same, as `PSI_API` are missing in many places.

Reply:
OK, I see. I think the way to go is to then put `PSI_API` in the header file. Using pybind11 will add `-fvisibility=hidden -fvisibility-inlines-hidden` to the compiler flags for `core.so`, which should give on Linux the same behavior as the default on MSVC.